holoviews/element/util.py

```diff
@@ -30,3 +39,60 @@ def compute_edges(edges):
         raise ValueError('Centered bins have to be of equal width.')
     edges -= width/2.
     return np.concatenate([edges, [edges[-1]+width]])
+
+
+def reduce_fn(x):
+    """
+    Aggregation function to get the first non-zero value.
+    """
+    values = x.values if pd and isinstance(x, pd.Series) else x
+    for v in values:
+        if not is_nan(v):
+            return v
+    return np.NaN
+
+
+def get_2d_aggregate(obj):
+    """
```

Comment:
Perhaps this would be better expressed as an operation? Then maybe it could have a minimal docstring example in the class docstring?